### ISSUE TITLE
Parsing variables to JSON in TransactionalEmail

### ISSUE THREAD

**Issue by cristiammercado** [mailjet/mailjet-apiv3-java#149]
Hi, 

Like I described in a past issue #147, I have just migrated the mailjet-client library to version `5.1.0` in my project to use `TransactionalEmail`. I created a `Map <String, String>` with some variables for a template created in the web application. But when parsing the object to JSON, Additional quotes are added to the strings. This problem didn't happen when using implementation with `JSONObject` and `MailjetRequest`.

Example with `TransactionalEmail`:

```java
Map<String, String> variables = new HashMap<>();
variables.put(EmailConstants.EMAIL_DATE, "a string text");
variables.put(EmailConstants.EMAIL_YEAR, "another text");
variables.put(EmailConstants.EMAIL_ID, "a last text example");

TransactionalEmail message = TransactionalEmail
  .builder()
  .to(new SendContact(email.toEmail(), email.toName()))
  .from(new SendContact(email.fromEmail(), email.fromName()))
  .templateID(email.templateId())
  .templateLanguage(true)
  .priority(email.priority())
  .customID(email.id())
  .variables(variables)
  .trackOpens(TrackOpens.ENABLED)
  .trackClicks(TrackClicks.ENABLED)
  .build();
```

Body sent to API (notice in the `variables` object the additional escaped quotes):

```json
{
  "Messages": [
    {
      "Cc": [],
      "TrackClicks": "enabled",
      "Bcc": [],
      "Priority": 3,
      "Headers": {},
      "From": {
        "Email": "xxxxx@xxxxxxxxxxxx.com",
        "Name": "xxxxxxxx"
      },
      "Attachments": [],
      "TemplateLanguage": true,
      "TrackOpens": "enabled",
      "Variables": {
        "E_DATE": "\"a string text\"",
        "E_YEAR": "\"another text\"",
        "E_MAIL_ID": "\"a last text example\"",
      },
      "CustomID": "8c0725fa-403c-496e-ac7e-xxxxxxxxx",
      "InlinedAttachments": [],
      "To": [
        {
          "Email": "xxxxxxxx@xxxxxxx.com",
          "Name": "xxxxxxxx xxxxxxxxxx"
        }
      ],
      "TemplateID": 1234567
    }
  ],
  "AdvanceErrorHandling": false,
  "SandboxMode": false
}
```

Same body in old  `JSONObject` and `MailjetRequest`: 

 ```json
{
  "Messages": [
    {
      "Cc": [],
      "TrackClicks": "enabled",
      "Bcc": [],
      "Priority": 3,
      "Headers": {},
      "From": {
        "Email": "xxxxx@xxxxxxxxxxxx.com",
        "Name": "xxxxxxxx"
      },
      "Attachments": [],
      "TemplateLanguage": true,
      "TrackOpens": "enabled",
      "Variables": {
        "E_DATE": "a string text",
        "E_YEAR": "another text",
        "E_MAIL_ID": "a last text example",
      },
      "CustomID": "8c0725fa-403c-496e-ac7e-xxxxxxxxx",
      "InlinedAttachments": [],
      "To": [
        {
          "Email": "xxxxxxxx@xxxxxxx.com",
          "Name": "xxxxxxxx xxxxxxxxxx"
        }
      ],
      "TemplateID": 1234567
    }
  ],
  "AdvanceErrorHandling": false,
  "SandboxMode": false
}
```

Thanks in advance!

**Comment by goacoustic** [comment]
Hi @cristiammercado,

Thank you for pointing this out.
I've fixed it in v5.1.1, could you please try it?

**Comment by cristiammercado** [comment]
Hi @goacoustic ,

Everything worked fine in version `5.1.1` 

Thank you!
